tests/test_handlers.py

```diff
@@ -53,3 +58,34 @@ def test_all_handlers(input_dir: Path, output_dir: Path, tmp_path: Path):
     except subprocess.CalledProcessError as exc:
         runnable_diff_command = " ".join(diff_command)
         pytest.fail(f"\nDiff command: {runnable_diff_command}\n{exc.stdout}\n")
+
+
+@pytest.mark.parametrize(
+    "handler",
+    (
+        pytest.param(handler, id=handler.NAME)
+        for handler_map in _ALL_MODULES_BY_PRIORITY
+        for handler in handler_map.values()
+    ),
+)
+def test_missing_handlers_integrations_tests(handler):
+    handler_name = handler.NAME
+    handler_path = (
+        Path(inspect.getfile(handler.__class__))
+        .relative_to(HANDLER_MODULE_PATH)
+        .with_suffix("")
+    )
+
+    if handler_name == handler_path.name:
```

Comment:
I know I was the one pushing for this 😄, but maybe I would not do this, because if we add new tests next to the existing, the files have to move. Which would cause unnecessary diffs in git commits. With Git LFS, we should embrace producing more test files with edge cases. Test directory structure would also be more consistent with no exception.